cmd/serve/serve.go

```diff
@@ -19,18 +22,12 @@ type Options struct {
 }
 
 func NewServeCmd() *cobra.Command {
-
 	opts := Options{
 		Development: false,
 	}
 
 	serverOpts := server.DefaultOptions()
-
-	healthEnabled := true
```

Comment:
nitpick: the reasoning the `enabled` option was placed in the `serve` package, and not in the `health` package, was to separate the setup of the servers from the health server itself.

Reply:
The settings are still set in the command serve via the flag, it is just the defaults that are in the health package. Don't you think it makes sense?

Reply:
It depends on the responsibility of the health package. I was looking at it as a provider of health server, so it was up to the caller (the serve command) to define if it would instantiate it or not.  But this discussion is not needed now.